eng/common/scripts/Prepare-Release.ps1

```diff
@@ -109,11 +109,15 @@ else
 $releaseDateString = $ParsedReleaseDate.ToString("MM/dd/yyyy")
 $month = $ParsedReleaseDate.ToString("MMMM")
 
-Write-Host
 Write-Host "Assuming release is in $month with release date $releaseDateString" -ForegroundColor Green
+if (Test-Path "Function:GetExistingPackageVersions")
+{
+    $releasedVersions = GetExistingPackageVersions -PackageName $packageProperties.Name -GroupId $packageProperties.Group
+    $latestReleasedVersion = $releasedVersions[$releasedVersions.Count - 1]
```

Comment:
Since strict mode is set for this script, you will get a failure here if `$releasedVersions` is empty: `OperationStopped: Index was outside the bounds of the array.`. Unless you want to explicitly fail here when empty, this is safer:

```suggestion
    $latestReleasedVersion = $releasedVersions | Select -Last 1
```